### PR TITLE
fix buildBankAccounts when have two or more STMTRS inside

### PR DESCRIPTION
**Pull request by jmarcelocjr**
The latest OFX that i received from Banco do Brasil come with two STMTRS and when the Ofx build the bank accounts he only get the first one. 
With that change he do a foreach into STMTRS to get all bank account that may be inside there.

Nothing  changes on return.

Also added the OFX to tests.

### CODE REVIEW

File: lib/OfxParser/Ofx.php [hunk 1]
```diff
@@ -147,32 +147,47 @@ private function buildBankAccounts(SimpleXMLElement $xml)
...
+            foreach ($accountStatement->STMTRS as $statementResponse) {
+                $bankAccounts[] = $this->buildBankAccount($accountStatement->TRNUID, $statementResponse);
+            }
         }
         return $bankAccounts;
     }
 
     /**
-     * @param SimpleXMLElement $xml
+     * @param $transactionUid
```
Comment (asgrim) [inline lib/OfxParser/Ofx.php]: Please document the type of `$transactionUid`

**Comment by jmarcelocjr** [comment]
@asgrim Thank you por your review!

added the type!

**Comment by asgrim** [comment]
Thanks @jmarcelocjr! :+1:

**Comment by asgrim** [comment]
Released in https://github.com/asgrim/ofxparser/releases/tag/1.2.2 - thanks!

**Pull request merged**